### ISSUE TITLE
Any label name

### ISSUE THREAD

**Issue by qingdao05** [SalesforceLabs/record-hunter#10]
If multiple items of the same object refer to a trading partner, the same label name is displayed for both items, and it is
not clear which name item is displayed.
Is it possible to set an arbitrary label name for the item name?
(For example, if you write 「FiledName as LabelName」)

**Comment by dmhoffritz** [comment]
Agreed. This is the only thing it's missing and would be extremely helpful.
